game/core/package.json

```diff
@@ -72,7 +72,8 @@
         "uuid": "^10.0.0"
     },
     "devDependencies": {
-        "@parcel/config-default": "^2.12.0",
+        "@babel/preset-typescript": "^7.24.7",
```

Comment:
Does it still give a warning on build?

Reply:
messed up with the changes for the file, thanks for pointing!